cypress/support/commands.ts

```diff
@@ -19,24 +19,33 @@ Cypress.Commands.add('injectAxe', () => {
 });
 
 // Needed for https://github.com/Bkucera/cypress-plugin-tab/issues/46
-Cypress.Commands.overwrite('visit', (originalFn, url, options = {}) => {
-  if (typeof url === 'object') {
-    // eslint-disable-next-line no-param-reassign
-    url = options.url;
+Cypress.Commands.overwrite('visit', (originalFn, urlOrOptions, inputOptions = {}) => {
```

Comment:
Changes in this file were from Cypress adding type defintions to `Cypress.Commands.add` and `Cypress.Commands.overwrite` in [Cypress v9](https://docs.cypress.io/guides/references/changelog#9-0-0). In most cases, this is an improvement. In the case of commands that use Typescript overrides, this causes problems. I've rearranged command overrides and added some `any` in cases where the Cypress types introduce problems that cannot be resolved.